ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -485,6 +485,82 @@ func (e *ExecutableHandlerImpl) validate() bool {
 	return true
 }
 
+func executeScript(script string) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	// Execute the shell script
+
+	cmd := exec.Command("sh", "-c", script)
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		log.WithFields(log.Fields{
+			"err":    err,
+			"stdout": stdout.String(),
+			"stderr": stderr.String(),
+		}).Errorf("error while executing ")
+	}
+
+	// Print the output from the script
+	fmt.Println("DONE")
+}
+
+/*
+* The AddFxpRules and RemoveFxpRules are two functions added as a workaround
+* to configure the FXP with point to point rules between the VFs initialised on a single
+* host. These rules assume that no NF has been deployed on the FXP.
+*
+* Function AddFxpRules will create all the point to point rules between all the initilised
+* VFs on the host.
+* Function RemoveFxpRules will remove these rules from the FXP.
+ */
+func AddFxpRules() {
+	executeScript(`
+IFS=$'\n' read -r -d '' -a vf_mac_addresses < <(ssh -o StrictHostKeyChecking=no -o ConnectTimeout=10 root@192.168.0.1 "/usr/bin/cli_client -cq" \
+    | awk '{if(($4 == "0x0") && ($6 == "yes")) {print $17}}')
+
+for src_mac in "${vf_mac_addresses[@]}"; do
+    for dst_mac in "${vf_mac_addresses[@]}"; do
+        smac=$(echo "$src_mac" | tr -d ':')
+        if [ "$src_mac" != "$dst_mac" ]; then
+            dmac=$(echo "$dst_mac" | tr -d ':')
+            vsi=$(echo "$src_mac" | awk -F: '{print $2}')
```

Comment:
I think it's good idea to install gawk in the Dockerfile for this awk to work in container environment

Reply:
Awk removed - handling this in go